internal/test/wsu/wsu_test.go

```diff
@@ -39,6 +39,9 @@ var (
 	ubi8Image = "registry.access.redhat.com/ubi8/ubi:latest"
 )
 
+const hybridOverlayLogPath = "C:\\k\\log\\hybridOverlay"
```

Comment:
`LogPath` seems to imply it could be a path for a single log file. What is the downside of using `hybridOverlayDir`?

Please use `hybrid-overlay` for the directory name.